orc/src/main/java/org/apache/iceberg/orc/OrcFileAppender.java

```diff
@@ -83,37 +81,13 @@ public void add(D datum) {
         batch.reset();
       }
     } catch (IOException e) {
-      throw new RuntimeException("Problem writing to ORC file " + path, e);
+      throw new RuntimeException("Problem writing to ORC file " + file.location(), e);
```

Comment:
This should be a `RuntimeIOException`. Looks like a mistake from the initial commit.